libs/vc-helper/src/main/java/uk/gov/di/ipv/core/library/vchelper/VcHelper.java

```diff
@@ -37,9 +37,17 @@ public static void setConfigService(ConfigService configService) {
         VcHelper.configService = configService;
     }
 
-    private static Set<String> getNonEvidenceCredentialIssuers() {
+    public static Set<String> getNonEvidenceCredentialIssuers() {
+        return VcHelper.getNonEvidenceCredentialIssuers(configService);
+    }
+
+    public static Set<String> getNonEvidenceCredentialIssuers(ConfigService configService) {
```

Comment:
It looks like the only thing calling this now overloaded method is the version above that doesn't take a configService. Is this added for a reason? If not, we can just keep the original version and drop this new one.

Reply:
Done.